addons/hr_expense/models/hr_expense.py

```diff
@@ -151,13 +151,21 @@ def _compute_state(self):
             else:
                 expense.state = "done"
 
-    @api.depends('quantity', 'unit_amount', 'tax_ids', 'currency_id')
+    @api.depends( 'total_amount', 'quantity', 'unit_amount', 'tax_ids', 'currency_id')
     def _compute_amount(self):
         for expense in self:
             if expense.unit_amount:
-                expense.untaxed_amount = expense.unit_amount * expense.quantity
-                taxes = expense.tax_ids.compute_all(expense.unit_amount, expense.currency_id, expense.quantity, expense.product_id, expense.employee_id.user_id.partner_id)
-                expense.total_amount = taxes.get('total_included')
+                amount = expense.unit_amount
+                quantity = expense.quantity
+            else:
+                # the taxes should be "Included In Price", as the entered
+                # total_amount includes all the taxes already
+                # for the cases with total price, the quatity is always 1
```

Comment:
```diff
- quatity
+ quality
```
this seems very particular, i'm not too sure about 1. in this case maybe it's correct, but usually the quantity can be rather important (for example pricelist could work or not based on price)